ideasbox/views.py

```diff
@@ -15,9 +16,11 @@
 def index(request):
     contents = Content.objects.published()[:3]
     random_book = Book.objects.available().order_by('?').first()
+    random_doc = Document.objects.order_by('?').first()
```

Comment:
One of the specifics of Ideas Box server, is that we will be running with very small dataset and under very low load, so I think this will scale in our context :)